content/developer/tutorials/getting_started/02_setup.rst

```diff
@@ -12,55 +12,83 @@ Prepare the environment
 First, follow the :ref:`contributing/development/setup` section of the contributing guide to prepare
 your environment.
 
-.. important::
-   The following steps are intended only for Odoo employees. The mentioned repositories are not
-   accessible to third parties.
-
 By now, you should have downloaded the source code into two local repositories, one for `odoo/odoo`
-and one for `odoo/enterprise`. These repositories are set up to push changes to pre-defined shared
+and one for `odoo/enterprise`. These repositories are set up to push changes to pre-defined
 forks on GitHub. This will prove to be convenient when you start contributing to the codebase, but
-for the scope of this tutorial, we want to avoid polluting the shared repositories with training
-material. Let's then develop your own module in a third repository `technical-training-sandbox`.
-Like the first two repositories, it will be part of the `addons-path` that references all
-directories containing Odoo modules.
+for the scope of this tutorial, we want to avoid polluting them with training material. Let's then
+develop your own module in a third repository `odoo/tutorials`. Like the first two repositories,
+it will be part of the `addons-path` that references all directories containing Odoo modules.
+In this repository, we will create our first module!
+
+.. note::
+
+   This repository also already contains some bare modules that will be used later on during
+   Javascript tutorials.
```

Comment:
It will probably change several times in the future so let's be vague.
```suggestion
   This repository also already contains some bare modules that will be used in other tutorials.
```